guide/popular-topics/reactions.md

```diff
@@ -155,6 +155,112 @@ if (message.content === '!fruits') {
+:::
+
+### Removing all reactions
+
+Removing all reactions from a message is the easiest, the API allows you to do this through a single call. It can be done through the <branch version="11.x" inline> `message.clearReactions()` </branch><branch version="12.x" inline> `message.reactions.removeAll()` </branch> method. 
+
+<branch version="11.x">
+
+```js
+message.clearReactions().catch(error => console.error('Failed to clear reactions: ', error));
+```
+
+</branch>
+<branch version="12.x">
+
+```js
+message.reactions.removeAll().catch(error => console.error('Failed to clear reactions: ', error));
+```
+
+</branch>
+
+### Removing reactions by emoji
+
+Removing reactions by emoji is not as straightforward as clearing all reactions. The API does not provide a method for selectively removing reactions by emoji, it only allows you to remove a user from a specific reaction. This means you will have to get the users who reacted with that emoji, and loop through and remove each one of them.
+
+Reaction collections are keyed by <branch version="11.x" inline>`name:id`</branch><branch version="12.x" inline>`id`</branch> for custom emojis and by `name` for unicode emojis (represented by their unicode character, see [here](/popular-topics/reactions.html#unicode-emojis)). Once you have the key you can simply run a `.get()` on `message.reactions` to get the reaction representing the emoji you want.
+
+<branch version="11.x">
+
+<!-- eslint-skip -->
+```js
+const reaction = message.reactions.get('Thonk:484535447171760141');
+try {
+	for (const user of reaction.users.values()) {
+		await reaction.remove(user);
+	}
+} catch (error) {
+	console.error('Failed to remove reactions.');
+}
+```
+
+</branch>
+<branch version="12.x">
+
+<!-- eslint-skip -->
+```js
+const reaction = message.reactions.get('484535447171760141');
+try {
+	for (const user of reaction.users.values()) {
+		await reaction.users.remove(user);
+	}
+} catch (error) {
+	console.error('Failed to remove reactions.');
+}
+```
+
+</branch>
+
+The reason we use a `for... of` loop over something like `forEach()` is due to `forEach()`'s behavior for async operations. `forEach()` will send out all calls almost at once even if we await inside of the function. However, if we `await` inside of a `for... of` loop, it will wait for the previous reaction to go through, and we avoid spamming the API with a lot of calls at once.
+
+### Removing reactions by user
+
+Removing reactions by user is similar to what you did before. However, instead of iterating through users of a reaction, you will iterate through reactions which include a user. To do this you will get all reactions and filter based on whether the user has reacted. If you are not familiar with `Collection.filter()` and `Collection.has()` take the time to understand what they do and then come back.
```

Comment:
I think the last sentence could be a `::: tip` block, since that's usually what we do with these sort of notes. Links to the methods would be nice too.